win32ss/gdi/ntgdi/d3dkmt.c

```diff
@@ -0,0 +1,715 @@
+/*
+ * PROJECT:     ReactOS Display Driver Model
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     D3DKMT dxgkrnl callbacks
+ * COPYRIGHT:   Copyright 2023 Justin Miller <justin.miller@reactos.org>
+ */
+
+#include <win32k.h>
+#include <reactos/rddm/rxgkinterface.h>
+#include <debug.h>
+
+/*
+ * It looks like Windows saves all the function pointers globally inside win32k.
+ * Instead, we're going to keep it static to this file and keep it organized in struct
+ * we obtained with the IOCTL.
+ */
+static REACTOS_WIN32K_DXGKRNL_INTERFACE DxgAdapterCallbacks = {0};
+
+/*
+ * This looks like it's done inside DxDdStartupDxGraphics, but I'd rather keep this organized.
+ * Dxg gets start inevitably anyway it seems at least on vista.
+ */
+VOID
+DxStartupDxgkInt()
+{
+    DPRINT("DxStartupDxgkInt: Entry\n");
+    /*
+     * TODO: Let DxgKrnl know it's time to start all adapters, and obtain the win32k<->dxgkrnl interface via an IOCTRL. 
+     * https://jira.reactos.org/browse/CORE-20027
+     */
+}
+
+BOOLEAN
+APIENTRY
+NtGdiDdDDICheckExclusiveOwnership(VOID)
+{
+    // We don't support DWM at this time, excusive ownership is always false.
+    return FALSE;
+}
+
+NTSTATUS
+APIENTRY
+NtGdiDdDDIGetProcessSchedulingPriorityClass(_In_  HANDLE                                unnamedParam1,
+                                            _Out_ D3DKMT_SCHEDULINGPRIORITYCLASS        *unnamedParam2)
+{
+    return 1;
+}
+
+NTSTATUS
+APIENTRY
+NtGdiDdDDISetProcessSchedulingPriorityClass(_In_ HANDLE                                     unnamedParam1,
+                                            _In_ D3DKMT_SCHEDULINGPRIORITYCLASS             unnamedParam2)
```

Comment:
```suggestion
NtGdiDdDDISetProcessSchedulingPriorityClass(_In_ HANDLE unnamedParam1,
                                            _In_ D3DKMT_SCHEDULINGPRIORITYCLASS unnamedParam2)
```
Maybe you could consider putting the parameters in their own lines (like what was done in the header).